src/Pumukit/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -77,15 +77,15 @@ public function getConfigTreeBuilder()
             ->info('URL to process uploaded files. Ex: https://localhost/tus')
             ->end()
             ->scalarNode('inboxUploadLIMIT')
-            ->defaultValue(10)
+            ->defaultValue(1)
```

Comment:
@pnieto inboxUploadLimit should be 10

Reply:
This value corresponds to https://uppy.io/docs/tus/#limit-5 and It is used to establish how many concurrent uploads to the PuMuKIT inbox. I think the most suitable value is that there is only one concurrent upload to the platform.